demos/JWST/S4_template.ecf

```diff
@@ -1,9 +1,9 @@
 # Eureka! Control File for Stage 4: Generate Lightcurves 
 
 # Number of spectroscopic channels spread evenly over given wavelength range
-nspecchan       20          # Number of spectroscopic channels
-wave_min        2.4         # Minimum wavelength. Set to None to use the shortest extracted wavelength from Stage 3.
-wave_max        4.0         # Maximum wavelength. Set to None to use the longest extracted wavelength from Stage 3.
+nspecchan       2          # Number of spectroscopic channels
+wave_min        1.5         # Minimum wavelength. Set to None to use the shortest extracted wavelength from Stage 3.
+wave_max        4.5         # Maximum wavelength. Set to None to use the longest extracted wavelength from Stage 3.
```

Comment:
These parameters are only true for NIRSpec, while the old values were only for NIRCam. We should have a template for each of the instruments I guess

Reply:
Yeah, I was thinking about this and I'm not sure how valuable separate ones would be for Stage 4(outside of the testing .ecfs). The number of channels here is only two, so not really applicable for a real NIRSpec analysis anyways. I think once people have gone through the quickstart, they should get familiar with opening up the .ecfs and tailoring the parameter inputs to their specific dataset, instead of using Eureka like a black box.

Reply:
Initially I was opposed to updating all these templates as you've done, but I've changed my mentality and agree that the files in the demos folder should be tailored to the Quickstart Guide